compiler-rt/test/memprof/TestCases/dump_at_exit.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: %clangxx_memprof %s -o %t
+
+// RUN: %env_memprof_opts=print_text=true:log_path=stdout:dump_at_exit=false %run %t | count 0
```

Comment:
What is `count 0` here?

Reply:
`count N` will pass if it reads `N` lines from stdin. So this just checks that there is no output.

Reply:
Ah count is a utility we bundle with LLVM for testing. TIL.